src/common/Header/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+type Props = {
```

Comment:
for react components there is a built in function to props passing that gives you more utility out of the box
the type name is React.FunctionComponent or React.FC to which you can pass a generic type which reflects on props - ie.
```
type HeaderProps = {
title: string;
}

const Header: React.FC<HeaderProps> = ({ title }) => ...
```
results in the same type security, gives you a cleaner code and allows you to extract the props to a separate file whenever they get too big to keep it in the component